cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -1265,10 +1265,16 @@ void writeUpgradeStartTimeIfNotExist() {
    * Upgrade the active run records.
```

Comment:
Can you add a sentence saying it returns whether all active run records have been upgraded